docker-compose.yml

```diff
@@ -72,7 +72,32 @@ services:
       - db
       - dor-indexing-app
       - suri
+      - redis
 
+  dor-services-worker:
+    image: suldlss/dor-services-app:latest
+    ports:
+      - 3006:3000
```

Comment:
I wonder if we can exclude ports since the queue UI ought to be available via dor-services-app image.

Reply:
.... that may be what I was just trying to resolve. Thanks!